engine/sbom/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Viewing the SBOM for Docker images
+description: Viewing the Software Bill of Materials (SBOM) for Docker images
+keywords: Docker, sbon, Anchore, images, Syft, security
+toc_min: 1
+toc_max: 2
+---
+
+A Software Bill Of Materials (SBOM) is analogous to a packing list for a shipment; it’s all the components that make up the software, or were used to build it. For container images this includes the operating system packages that are installed (e.g.: ca-certificates) along with language specific packages that the software depends on (e.g.: log4j). The SBOM could include only some of this information or even more details, like the versions of components and where they came from.
+
+> :warning: *Note:* This command is considered _experimental_ and may change or be removed in the future.
+> Please leave feedback and report issues on [its repository](https://github.com/docker/sbom-cli-plugin).
```

Comment:
Suggest we add a section heading `## Feedback` and move this to the end of the topic.

```
## Feedback

Thanks for trying the Docker SBOM CLI plugin. We’d love to hear from you. You can provide feedback and report any bugs through the Issues tracker in the [docker/[sbom-cli-plugin](https://github.com/docker/sbom-cli-plugin){: target="_blank" rel="noopener" class="_"} GitHub repository.